### ISSUE TITLE
Loki: Grafana 9.0.0 strips path prefix from url

### ISSUE THREAD

**Issue by gjed** [grafana/grafana#50939]
**What happened**:
Grafana 9 strips loki datasource path prefix from url. 
For example, if url is `https://example.com/ciao`, all request to datasource are made as `https://example.com/loki/api/....`.
Therefore all results are errors

**What you expected to happen**:
If url is `https://example.com/ciao`, all request to datasource should be made as `https://example.com/ciao/loki/api/....`.

**How to reproduce it (as minimally and precisely as possible)**:
1. Have a loki instance, configured with :
```
server:
    http_listen_port: 9070
    http_path_prefix: /ciao
```
2. An apache (or something else) that reverse proxy the loki instance:

```
<VirtualHost *:443>
 ServerName example.com
 ...
 ProxyPass /ciao http://loki:9070/ciao disablereuse=On
 ProxyPassReverse /ciao http://loki:9070/ciao disablereuse=On
 ...
</VirtualHost>
``` 
4. Add loki datasource on grafana 9.0.0 with url `https://example.com/ciao`

**Anything else we need to know?**:
Tested with Grafana 9.0.0 the behavior is as above.
Tested with Grafana 8.5.4 the behavior is as expected.

**Environment**:
- Grafana version: 9.0.0 (docker)
- Data source type & version: Loki 2.3.0 (docker)
- OS Grafana is installed on: Dcoker, Debian 11
- User OS & Browser: Debian 11, Chrome 
- Grafana plugins: -
- Others:

**Comment by till** [comment]
Can reproduce this with 9.0.0.

**Comment by gabor** [comment]
thanks for the report, i can confirm the bug, we are working on a fix.

**Comment by gabor** [comment]
fixed, will be available in grafana 9.0.1
